pt/controller_1/comm.py

```diff
@@ -11,38 +18,44 @@ def __init__(self, func):
         self._password = get_stomp_password()
         self._port = get_stomp_port()
         self._ids = get_stomp_actuator_ids()
-        self.conn = stomp.Connection([(self._url, self._port)])
-        self.conn.set_listener('', self)
+        self.heartbeats = (
+            15000,
```

Comment:
please define these constants at the top of the file. Remember to give them (capitalized) names.